### ISSUE TITLE
Will not connect to ttrss, plugin will not enable

### ISSUE THREAD

**Issue by gbcox** [powerivq/ttrss-push-notifier-server#1]
Will not connect to url.  My url is https... is this not supported?

Also, when I try to enable the plugin in ttrss, it won't enable...

**Comment by powerivq** [comment]
@gbcox I use https without a problem. I'm not sure why it doesn't enable. Do you see an error message? Or didn't you click on "Enable selected plugins"?

**Comment by gbcox** [comment]
Yes, I tried to enable the plugin, I click on it, it turns green, when I then click to enable selected plugins, it deselects... other plugins enable fine.
